pinot-core/src/main/java/org/apache/pinot/core/data/manager/BaseTableDataManager.java

```diff
@@ -628,9 +628,15 @@ private File downloadAndStreamUntarWithRateLimit(String segmentName, SegmentZKMe
         maxStreamRateInByte);
     String uri = zkMetadata.getDownloadUrl();
     try {
-      File ret = SegmentFetcherFactory.fetchAndStreamUntarToLocal(uri, tempRootDir, maxStreamRateInByte);
-      LOGGER.info("Download and untarred segment: {} for table: {} from: {}", segmentName, _tableNameWithType, uri);
-      return ret;
+      try {
+        File ret = SegmentFetcherFactory.fetchAndStreamUntarToLocal(uri, tempRootDir, maxStreamRateInByte);
+        LOGGER.info("Download and untarred segment: {} for table: {} from: {}", segmentName, _tableNameWithType, uri);
+        return ret;
+      } catch (Exception e) {
+        _serverMetrics.addMeteredTableValue(_tableNameWithType,
+            ServerMeter.SEGMENT_TOTAL_STREAMED_DOWNLOAD_UNTAR_FAILURES, 1L);
```

Comment:
IMO, it's better to reuse the old metric and mark this pr as an incompatible change.

Reply:
+1 I would rather not introduce another metric.

Reply:
That is what I though, so to make sure we are in sync: we will redefine the old metric to tick on individual failures and may be just log when we exceed the max retries?

Reply:
Done in commit 84c64c1f925b9fabe62f51c9ca2a466414fff34d